third_party/move/move-model/bytecode/src/function_target.rs

```diff
@@ -90,6 +90,10 @@ pub struct FunctionData {
     pub ghost_type_param_count: usize,
     /// A map for temporaries to associated name, if available.
     pub local_names: BTreeMap<TempIndex, Symbol>,
+    /// Maps the attribute id of a bytecode to the ids of the source temporaries.
```

Comment:
What does this mean, 'id of a source temporary'? You should explain what this is. Also, I'm wondering why we are using attributes here in the first place, why not locations directly, similar as in the existing map?

Also those maps should be declared SxS with `locations`.